xiblint/rules/named_colors.py

```diff
@@ -36,6 +38,14 @@ def check(self, context):  # type: (XibContext) -> None
             if ignore_alpha and element.get('alpha') is not None and element.get('alpha') != '1':
                 continue
 
+            # Skip clear color (if configured)
+            if (
+                    allow_clear_color and
+                    element.get('alpha') is not None and element.get('white') is not None and
```

Comment:
Currently, when using "clear" as a color in xibs or storyboards Xcode translates that to `<color white="0.0" alpha="0.0"`, which is what this validations accomplish.

Lmk if I should only validate `alpha="0.0"` instead

Reply:
This is good, it should match whatever IB's `clearColor`